project.clj

```diff
@@ -102,7 +102,7 @@
   :figwheel {:http-server-root "public"
              :server-logfile "log/figwheel_server.log"
              :nrepl-port 7002
-             :css-dirs ["target/resources/public/css"]
+             :css-dirs ["target/resources/public/css/en" "target/resources/public/css/fi" "target/resources/public/css/sv"]
```

Comment:
Interesting that it's not recursive. `hawk` seems to support it but I think it's not used recursively here then.